aws/cloudformation-templates/web-ui-pipeline.yaml

```diff
@@ -139,7 +140,11 @@ Resources:
 
                 for obj in source_bucket.objects.filter(Prefix=source_path):
                     source = { 'Bucket': source_bucket_name, 'Key': obj.key }
-                    target = target_bucket.Object(obj.key)
+                    if len(resource_bucket_path)>0:
+                      # need to remove the resource_relative_path for the target images directory
+                      target = target_bucket.Object(obj.key[len(resource_bucket_path):])
```

Comment:
NIT: This works, I would prefer using resource_bucket_path.replace(resource_bucket_path, '')  rather than adding multiple conditions or using the length.

Reply:
I wanted to avoid a "replace" because if resource_bucket_path contains somethings that is included later in the path it will break.
For example if customer used images/ as a resource path.. 


I could get this in one-liner, but I kept a if/else for clarity